es3.0/html/indexflat.php

```diff
@@ -5,6 +5,22 @@
     <!-- Don't include accord.js -->
 </head>
 <body>
+  <script>
+    window.onload = function () {
+      try {
+        let params = new URL(document.location.toString()).searchParams;
+        let subpage = params.get("subpage");
+        if (subpage !== null) { 
+          const links = document.querySelectorAll('a');
```

Comment:
By just looking for the link in the DOM and clicking it, we should avoid having to whitelist anything. If it is referring to some link on the page, then the user can click it, if not, then it does not do anything.